src/app/components/route-block/index.js

```diff
@@ -33,196 +38,36 @@ export function RouteBlock() {
-								<div>
-									<div>
-										<h2>Attributes</h2>
-									</div>
-									<ul className="bb-route-block__attributes">
-										{ Object.entries(
-											blockType.attributes
-										).map( ( [ key, attributeScheme ] ) => (
-											<li key={ key }>
-												<strong>{ key }</strong>{ ' ' }
-												{ attributeScheme.type &&
-													`(${ attributeScheme.type })` }
-											</li>
-										) ) }
-									</ul>
-								</div>
-							</div>
-						);
-					}
-
-					if ( currentPanel.name === 'example' ) {
-						return (
-							<div className="bb-route-block__example">
-								<BlockPreview
-									viewportWidth={ 1000 }
-									blocks={ getBlockFromExample(
-										blockType.name,
-										{
-											attributes:
-												blockType.example.attributes,
-											innerBlocks:
-												blockType.example.innerBlocks,
-										}
-									) }
-								/>
-							</div>
-						);
-					}
+			<ButtonGroup>
+				{ tabs.map( ( tab, key ) => {
+					const destination =
+						tab.name === 'card'
+							? `/block/${ slug }`
+							: `/block/${ slug }/${ tab.name }`;
 
-					if ( currentPanel.name === 'markup' ) {
-						return (
-							<pre className="bb-route-block__markup">
-								{ serialize(
-									getBlockFromExample( blockType.name, {
-										attributes:
-											blockType.example.attributes,
-										innerBlocks:
-											blockType.example.innerBlocks,
-									} )
-								) }
-							</pre>
-						);
-					}
+					const isSelected =
+						tab.name === currentTab ||
+						( currentTab === undefined && tab.name === 'card' );
 
-					if ( currentPanel.name === 'editor' ) {
-						return (
-							<Editor
-								key={ blockType.name }
-								initialBlocks={ [
-									blockType.example
-										? getBlockFromExample( blockType.name, {
-												attributes:
-													blockType.example
-														.attributes,
-												innerBlocks:
-													blockType.example
-														.innerBlocks,
-										  } )
-										: createBlock( blockType.name ),
-								] }
-							/>
-						);
-					}
+					return (
+						<Link
+							key={ key }
+							className={ `components-button ${
+								isSelected ? 'is-pressed' : ''
+							}` }
+							to={ destination }
+						>
+							{ tab.title }
+						</Link>
+					);
+				} ) }
+			</ButtonGroup>
 
-					if ( currentPanel.name === 'stories' ) {
-						return (
-							<div className="bb-route-block__stories">
-								{ blockStories.map( ( blockStory ) => (
-									<div key={ blockStory.name }>
-										<h3>{ blockStory.name }</h3>
-										<BlockPreview
-											viewportWidth={ 1000 }
-											blocks={ blockStory.blocks.map(
-												( block ) =>
-													getBlockFromExample(
-														block.name,
-														block
-													)
-											) }
-										/>
-									</div>
-								) ) }
-							</div>
-						);
-					}
-				} }
-			</TabPanel>
+			<Tabs
```

Comment:
For me, this is probably badly named. if i see a `Tabs` component, I expect it to be some generic UI component not something rendering block tabs.

For the moment, I'd personally leave this inline here. If  we want to split things, I'd write a component per tab instead and leave the container here.

```
BlockCard
BlockExample
BlockEditor
BlockStories
```